virttest/remote.py

```diff
@@ -156,7 +156,8 @@ def handle_prompts(session, username, password, prompt, timeout=10,
                  r"[Pp]lease wait", r"[Ww]arning", r"[Ee]nter.*username",
                  r"[Ee]nter.*password", r"[Cc]onnection timed out", prompt,
                  r"Escape character is.*",
-                 r"Command>"],
+                 r"Command>",
+                 r"lost connection"],
```

Comment:
Do we need consider capital letter for lost connection, e.g. [Ll]ost [Cc]onnection ?

Reply:
@chunfuwen , ' [Cc]onnection' should not be necessary as it is after 'lost'. I updated with '[Ll]lost'. Thanks.